test/ret_test.exs

```diff
@@ -157,6 +161,147 @@ defmodule RetTest do
       assert 0 === count(HubRoleMembership, hub)
       assert 0 === count(RoomObject, hub)
     end
+
+    test "deletes account's avatars" do
+      {:ok, admin_account: admin_account} = create_admin_account("admin")
+      test_account = create_account("test")
+
+      [_avatar, avatar_owned_files] = create_avatar(test_account)
+
+      1 = count(Avatar, test_account)
+      7 = count(OwnedFile, test_account)
+      true = owned_files_exist?(avatar_owned_files)
+
+      assert :ok = Ret.delete_account(admin_account, test_account)
+
+      assert 0 === count(Avatar, test_account)
+      assert 0 === count(OwnedFile, test_account)
+      assert false === owned_files_exist?(avatar_owned_files)
```

Comment:
This might read a little nicer as `refute owned_files_exist?(avatar_owned_files)`